getting-started/con-configuring-dynamic-plugins-operator.adoc

```diff
@@ -0,0 +1,26 @@
+[id='con-configuring-dynamic-plugins-operator_{context}']
+= Configuring dynamic plugins with the Operator
+
+You can store the configuration for dynamic plugins in a ConfigMap object, which can then be referenced into the Custom Resource.
+
+== Example using the GitHub dynamic plugin
+In OpenShift, you can use the following content as a base template to create a ConfigMap named `dynamic-plugins-rhdh`:
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic'
+        disabled: false
+        pluginConfig: {}
+----
+
+[NOTE]
+If the `pluginConfig` field references environment variables, they will need to be provided by including the variables in your `secrets-rhdh` secret.
```

Comment:
"they will need to be provided by including the variables in your `secrets-rhdh` secret" => "they shall be defined in your `secrets-rhdh` secret".